src/UI/Seller/src/assets/appConfigs/defaultadmin-test.json

```diff
@@ -3,9 +3,9 @@
   "sellerID": "my-seller-id",
   "sellerName": "Default Admin",
   "appname": "Default Admin",
-  "clientID": "xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx",
-  "middlewareUrl": "https://my-default-admin-test.com",
-  "translateBlobUrl": "https://my-translation-files/ngx-translate/i18n/",
-  "blobStorageUrl": "https://myblobstoragecontainer.blob.core.windows.net",
+  "clientID": "93268419-E864-444E-8BB1-1EB1B57C530B",
```

Comment:
Confirming this should have a specific `clientID?`

Reply:
Good catch. Updated